currency_rate_update/views/res_currency_rate.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+    Copyright 2019 Brainbean Apps (https://brainbeanapps.com)
+    License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+-->
+<odoo>
+    <record model="ir.ui.view" id="view_currency_rate_form">
+        <field name="name">res.currency.rate.form</field>
+        <field name="model">res.currency.rate</field>
+        <field name="inherit_id" ref="base.view_currency_rate_form" />
+        <field name="arch" type="xml">
+            <form position="inside">
+                <div class="oe_chatter">
```

Comment:
this div should be replaced with <chatter />

Reply:
other than this it looks fine.

Reply:
Thanks, it's already fixed